web/app/view/CommandDialogController.js

```diff
@@ -113,9 +113,18 @@ Ext.define('Traccar.view.CommandDialogController', {
         });
     },
 
+    onTextChannelChange: function (checkbox, newValue) {
+        var typesStore = this.lookupReference('commandType').getStore();
+        typesStore.getProxy().setExtraParam('textChannel', newValue);
+        typesStore.reload();
+    },
+
     onSendResult: function (options, success, response) {
         if (success) {
-            Ext.toast(Strings.commandSent);
+            Ext.toast({
+                html: Strings.commandSent,
+                align: 'br'
```

Comment:
Default behavior was changed in ExjJS 6.20 and we decided to revert it back to bottom-right https://github.com/tananaev/traccar-web/pull/355